CONTRIBUTING.md

```
@@ -33,6 +33,194 @@ Before contributing, ensure you have the following installed:
+
+- **Description**: The base URL for IDX (Indexing Service).
+
+#### `NEXT_PUBLIC_RELEASE_VERSION`
+
+- **Description**: The current release version of the application.
+
+#### `NEXT_PUBLIC_SENTRY_DSN`
+
+- **Description**: The Data Source Name (DSN) for Sentry error tracking.
+
+### Backend
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)